lib/util/gh.js

```diff
@@ -2,18 +2,18 @@
 
 module.exports = gh
 
+const {URL} = require('url')
+
 // Return a URL to GitHub, relative to an optional `repo` object, or `user` and
 // `project`.
-function gh(repo, project) {
-  var base = 'https://github.com/'
-
+function gh(base, repo, project) {
   if (project) {
     repo = {user: repo, project: project}
   }
 
   if (repo) {
-    base += repo.user + '/' + repo.project + '/'
+    return new URL('/' + repo.user + '/' + repo.project, base) + '/'
```

Comment:
```suggestion
    return base + '/' + repo.user + '/' + repo.project + '/
```

Reply:
You sure? Have you seen [my comment](https://github.com/remarkjs/remark-github/pull/24#discussion_r485018334) about trailing slashes?

Reply:
Sorry I missed this one!

I dunno, I feel that that’s a failure by users and should result in an error / that output?